lib/matplotlib/tests/test__style_helpers.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+import matplotlib.colors as mcolors
+from matplotlib.lines import _get_dash_pattern
+from matplotlib._style_helpers import style_generator
+
+
+@pytest.mark.parametrize('key, value', [('facecolor', ["b", "g", "r"]),
+                                        ('edgecolor', ["b", "g", "r"]),
+                                        ('hatch', ["/", "\\", "."]),
+                                        ('linestyle', ["-", "--", ":"]),
+                                        ('linewidth', [1, 1.5, 2])])
+def test_style_generator_list(key, value):
```

Comment:
```suggestion
def test_style_generator_list(key, value):
    """Test that style parameter lists are distributed to the generator."""
```
or as similar wording.